app/domain/authentication/authn_k8s/k8s_resource_validator.rb

```diff
@@ -0,0 +1,50 @@
+module Authentication
+  module AuthnK8s
+
+    class K8sResourceValidator
```

Comment:
@abrahamko can you please write a comment here saying why you chose to use a regular class `K8sResourceValidator` that has a `validate` method instead of using a command class `ValidateK8sResource`?
I'm not saying it is wrong but i would like @jonahx 's opinion on this so your line of thought will help give him context.

Reply:
done